ledger/apply/asset.go

```diff
@@ -134,6 +138,10 @@ func AssetConfig(cc transactions.AssetConfigTxnFields, header transactions.Heade
 		if err != nil {
 			return err
 		}
+		err = balances.DeallocateAsset(creator, cc.ConfigAsset, false)
+		if err != nil {
+			return err
+		}
```

Comment:
double call does not look good, how about adding bit flags specifying global or local, or both.

Reply:
What's wrong with double call? Asset params and asset holdings are modified separately in account data.

Reply:
The `Balance` interface is simpler as is.

Reply:
Calling twice is a clear indication that the interface is wrong.. I'd agree with Pavel that we should update the function call.